packages/expo-video/src/VideoView.tsx

```diff
@@ -21,15 +25,15 @@ export class VideoView extends PureComponent<VideoViewProps> {
   /**
    * Enters fullscreen mode.
    */
-  enterFullscreen() {
-    this.nativeRef.current?.enterFullscreen();
+  async enterFullscreen() {
```

Comment:
Since the function now has a `return`, explicitly specifying the returned type might be a good idea.